docs/src/reference-guide/components/graphql/directives.md

```diff
@@ -1,18 +1,16 @@
 # Directives
 
-> Per GraphQL: A directive is an identifier preceded by a @ character, optionally followed by a list of named arguments, which can appear after almost any form of syntax in the GraphQL query or schema languages.
+> Per GraphQL: A directive is a keyword preceded by a @ character (optionally followed by a list of named arguments) which can appear after almost any form of syntax in the GraphQL query or schema languages.
 
-- As of this writing, the list of supported Fuel GraphQL schema directives includes:
+As of this writing, the list of supported Fuel GraphQL schema directives includes:
 
-  - `@indexed`
-  - `@unique`
-  - `@join`
-
-Using our `Library` and `Book` example from the previous [Foreign Keys](../database/foreign-keys.md) section -- given the following schema:
+- `@indexed`
+- `@unique`
+- `@join`
 
 ## `@indexed`
 
-The `@indexed` directive adds an index to the underlying database column for the indicated field of that type. Generally, an index is a data structure that allows you to quickly locate data without having to search each row in a database table.
+The `@indexed` directive adds a database index to the underlying column for the indicated field of that type. Generally, a database index is a data structure that allows you to quickly locate data without having to search each row in a database table.
 
```

Comment:
this is referring to postgres indexes, I think it's worth explicitly noting that this has nothing to do with a fuel index, or link to this [https://www.postgresql.org/docs/current/indexes.html](), so a user could see exactly what this is and quickly understand the difference.